### ISSUE TITLE
Alt text is not descriptive [../tools/console files]

### ISSUE THREAD

**Issue by timngyn** [aws-amplify/docs#7069]
**Describe the content issue**:
Images listed below do not have descriptive/informative alt text. For markdown the alt text is located between the square brackets, `![alt text](...)`.

The list below isn't exhaustive since I found these images based on the word count for the alt text, but it should be a good starting point to make our alt text better for people who use screen readers.

Some resources to help write alt text:
- https://webaim.org/techniques/alttext/
- https://www.a11yproject.com/posts/are-you-making-these-five-mistakes-when-writing-alt-text/

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/tools/console/index.mdx:
```
   93  
   94: ![GIF showing data modeling](/images/console/datamodel.gif)
   95  

   99  
  100: ![Screenshot showing Figma to Code](/images/console/ui-figmatocode.png)
  101  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/tools/console/storage/file-browser/index.mdx:
```
  50  
  51: ![Amplify Studio file browser](/images/console/fileBrowser.png)
  52  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/tools/console/storage/file-storage/index.mdx:
```
  49  
  50: ![Amplify Studio storage start page](/images/console/storageStart.png)
  51  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/tools/console/tutorial/bindui/index.mdx:
```
  56  
  57: ![Video of Shuffle Preview](/images/studio/shuffle-preview.gif)
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/tools/console/tutorial/buildui/index.mdx:
```
   96  
   97: ![edit button](/images/studio/buildui/edit-ui-library.png)
   98  

  100  
  101: ![selection](/images/studio/buildui/multiple-components-selection.png)
  102  

  104  
  105: ![delete](/images/studio/buildui/delete-components.png)
```

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/tools/console/tutorial/code/index.mdx:
```
  59  
  60: ![GIF showing responsive app](/images/studio/responsive.gif)
  61  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/tools/console/tutorial/collections/index.mdx:
```
  57  
  58: ![Screenshot of collection stylings applied](/images/studio/collection-layout.png)
  59  

  65  
  66: ![Screenshot of collection data applied](/images/studio/collection-data.png)
  67  
```
